src/AppBundle/Controller/RfXVoteCalculatorController.php

```diff
@@ -0,0 +1,228 @@
+<?php
+/**
+ * This file contains the code that powers the RfX Vote Calculator page of xTools.
+ *
+ * @category RfXVoteCalculator
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfap
```

Comment:
Per above I think we should remove author and link here and in the other files. License is OK but I'd argue it is redundant as the license does not vary file to file within our own code